lib/health-check/health-check.decorator.ts

```diff
@@ -0,0 +1,38 @@
+import type * as Swagger from '@nestjs/swagger';
+import { getHealthCheckSchema } from './health-check.schema';
+
+const noop = () => {};
+
+function getSwaggerDefinitions(swagger: typeof Swagger) {
```

Comment:
Instead of using a top-level `import type * as` you can also use this: `swagger: typeof import('@nestjs/swagger')`

Reply:
done!